Source/Runtime/Measurement Server.lvproj

```diff
@@ -68,71 +68,43 @@
 				<Item Name="JDP Timestamp.lvlib" Type="Library" URL="/&lt;vilib&gt;/JDP Science/JDP Science Common Utilities/Timestamp/JDP Timestamp.lvlib"/>
 				<Item Name="Get LV Class Name.vi" Type="VI" URL="/&lt;vilib&gt;/Utility/LVClass/Get LV Class Name.vi"/>
 				<Item Name="Base64 Support.lvlib" Type="Library" URL="/&lt;vilib&gt;/JDP Science/JDP Science Common Utilities/Base64/Base64 Support.lvlib"/>
-				<Item Name="Simple Error Handler.vi" Type="VI" URL="/&lt;vilib&gt;/Utility/error.llb/Simple Error Handler.vi"/>
```

Comment:
Is all this stuff supposed to be removed from here?

Reply:
I don't know. I feel like these VIs randomly appear and disappear. I'm inclined to just trust that LabVIEW knows what it's doing.

Reply:
They are in the dependencies section, so I agree with @pbirkhol-ni here that these should be managed by LabVIEW.

Reply:
Yeah, it's annoying that the automatically calculated list of dependencies by LV is persisted in the project file such that it creates noise when trying to view "real" edits made to the project file. It can sometimes be useful if you see something from a package show up that you don't expect to depend on. However, it's mostly just noise.